src/Containers/Tag.tsx

```diff
@@ -6,15 +6,17 @@ import { Main } from '@Utils/BaseStyles';
 
 export interface TagProps {
     icon?: React.ForwardRefExoticComponent<React.RefAttributes<SVGSVGElement>>;
+    onClick?: () => void;
```

Comment:
I know this is the way I do it, but look into extending TagProps with HTMLDivElement props, that would be better